src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java

```diff
@@ -189,6 +191,17 @@ public void populateValues(
 		ResultSetMetaData m = rs.getMetaData();
 		int numColumns = m.getColumnCount();
 
+
+		// organize dimensions by name
+		Map< String, IDimension> dimensionsMap = new HashMap< String, IDimension> (); 
+		for( IDimension dimension : dimensions ) 
+			dimensionsMap.put( dimension.getName(), dimension );
+
+		// organize variables by name
+		Map< String, IVariableEncoder> encodersMap = new HashMap< String, IVariableEncoder> (); 
```

Comment:
I think renaming encoders to variables above made the code much more readable - perhaps consider doing the same here?